admin/etc/module.xml

```diff
@@ -6,5 +6,9 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Pronko_LiqPayAdmin" />
+    <module name="Pronko_LiqPayAdmin">
+        <sequence>
+            <module name="Magento_Sales"/>
```

Comment:
I don't think we should add sequence for Sales module

Reply:
Fixed